systemtest/src/main/java/io/strimzi/systemtest/enums/ClusterOperatorInstallType.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.enums;
+
+public enum ClusterOperatorInstallType {
+    OLM,
+    HELM,
+    BUNDLE
```

Comment:
I know this is something you didn't introduced. But any chance we can change here `BUNDLE` to something like `YAML`? I do not think bundle is really clear to what it means. Or is it more complicated then just changing the Enum? (e.g. some pipelines etc.)

Reply:
well yes you need to also replace it in pipelines :)

Reply:
Hmm, ok. Then lets not make this PR more complicated :-(.

Reply:
I will fix that in different PR :)

Reply:
Even if in different other repos I saw the YAMLs being referred as bundle. For example, installation of the prometheus-operator provides a `bundle.yaml` containing all the resources.

Reply:
@ppatierno I do not think it is a _wrong_ name per se. The problem is that it is ambiguous name. Lot of things are being referred to as _bundle_ ... among others the OLM files are bundles as well. That is why this is so confusing.